internal/proxy/task_index_test.go

```diff
@@ -651,6 +658,7 @@ func Test_parseIndexParams(t *testing.T) {
 		}
 		err := cit.parseIndexParams()
 		assert.NoError(t, err)
+		sortKeyValuePairs(cit.newIndexParams)
 		assert.Equal(t, cit.newIndexParams, []*commonpb.KeyValuePair{{Key: common.IndexTypeKey, Value: indexparamcheck.IndexHybrid},
```

Comment:
just use `assert.ElementsMatch` may be better?

Reply:
good method, this is also ok